### ISSUE TITLE
Import failed

### ISSUE THREAD

**Issue by Kewth** [LingDong-/wenyan-lang#480]
I try to run `wenyan examples/import.wy` to make a test, only to get an error:

```plain
/usr/lib/node_modules/@wenyanlang/cli/index.min.js:36524
    if (imports[i] in lib[lang]) {
                   ^

TypeError: Cannot use 'in' operator to search for '算經' in undefined
    at extractMacros (/usr/lib/node_modules/@wenyanlang/cli/index.min.js:36524:20)
    at compile (/usr/lib/node_modules/@wenyanlang/cli/index.min.js:1726:16)
    at getCompiled (/usr/lib/node_modules/@wenyanlang/cli/index.min.js:2112:10)
    at exec (/usr/lib/node_modules/@wenyanlang/cli/index.min.js:2203:16)
    at Object.<anonymous> (/usr/lib/node_modules/@wenyanlang/cli/index.min.js:2079:3)
    at __webpack_require__ (/usr/lib/node_modules/@wenyanlang/cli/index.min.js:31:30)
    at /usr/lib/node_modules/@wenyanlang/cli/index.min.js:95:18
    at /usr/lib/node_modules/@wenyanlang/cli/index.min.js:98:10
    at webpackUniversalModuleDefinition (/usr/lib/node_modules/@wenyanlang/cli/index.min.js:4:20)
    at Object.<anonymous> (/usr/lib/node_modules/@wenyanlang/cli/index.min.js:11:3)   

```

Even I try to clone the source, running `npm run test` sucessfully.

`node --version` v13.5.0

`uname -a` Linux kewth-computer 5.3.15-1-MANJARO #1 SMP PREEMPT Thu Dec 5 11:01:29 UTC 2019 x86_64 GNU/Linux

**Comment by antfu** [comment]
Hi @Kewth. What's the cli version you use? It seems to work on my machine. You can check the version by
```
wenyan -v
```
Thanks

**Comment by Kewth** [comment]
@antfu the version is:

`wenyan -v` 0.2.0

**Comment by antfu** [comment]
@Kewth This is a bug in v0.2.0. It should be fixed in v0.2.1. Please update and check it out. Thanks for the feedback!

**Comment by Kewth** [comment]
Thanks, it work well now.
